configure.ac

```diff
@@ -766,6 +763,9 @@ dnl ***************************************************************************
 
 dnl check OpenSSL static linking
 PKG_CHECK_MODULES(OPENSSL, openssl >= $OPENSSL_MIN_VERSION,, OPENSSL_LIBS="")
```

Comment:
I think we should give some details about this decision to users who don't have OpenSSL libs and want to compile syslog-ng. ( I mean this is not a simple configure error but a feature :-) )

I'll set `AC_MSG_ERROR` as `action-if-not-found` in `PKG_CHECK_MODULES(...)`